mappings/net/minecraft/client/world/FlatWorldGeneratorPresets.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/unmapped/C_brlntonw net/minecraft/client/world/FlatWorldGeneratorPresets
+	METHOD m_qyjelmkr register (Ljava/lang/String;)Lnet/minecraft/unmapped/C_xhhleach;
+		ARG 0 id
+	METHOD m_sybrxjpq getHolder ()Lnet/minecraft/unmapped/C_cjzoxshv;
```

Comment:
Isn't this `bootstrap` in mojmap?

Reply:
Yes, but i didnt like it since it doesn't always bootstrap, only the first time